lib/estimator/estimator.rb

```diff
@@ -0,0 +1,89 @@
+module Estimator
+  class Estimate < ActiveRecord::Base
+    require 'active_shipping'
+
+    def self.query(ship_params)
+      @api_call_ok = true
+      ship_array = shipment_array(ship_params)
+      ups_rates = get_rates(ship_array, ups)
+      usps_rates = get_rates(ship_array, usps)
+      quote = assemble_quote(ups_rates, usps_rates)
+      if @api_call_ok
+        return quote
+      else
+        return nil
+      end
+      @api_call_ok = true
+    end
+
+    def self.ups
+      ActiveShipping::UPS.new(login: 'shopifolk', password: 'Shopify_rocks', key: '7CE85DED4C9D07AB')
+    end
+
+    def self.usps
+      ActiveShipping::USPS.new(login: '677JADED7283')
+    end
+
```

Comment:
It's good practice to start creating .env files to store login info and keys to make API calls. Even for fake ones! Make a habit out of it!